.github/workflows/zscaler-iac-scan.yml

```
@@ -28,6 +28,7 @@ permissions:
 
 jobs:
   zscaler-iac-scan:
+    if: ${{ secrets.ZSCANNER_CLIENT_ID != '' && secrets.ZSCANNER_CLIENT_SECRET != '' }}
```

actionlint (1.7.11)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)